### DISCUSSION TITLE
Unable to Listen any Event

### DISCUSSION THREAD

**Discussion by adeel94mirza** [spatie/laravel-webhook-server#88] [Q&A]
Laravel **v7.10.3**
spatie/laravel-webhook-server **v1.12.0**
php **v7.4.3**

I am successfully able to send webhook but unable to listen to any events fired afterwards.

I have registered the events and listeners in the EventServiceProvider as well as turned on automatic discovery of Events/Listeners in Laravel

**Comment by DannyLMD** [comment]
Laravel **v11.24.1**
spatie/laravel-webhook-server **v3.8.3**
php **v8.3.17**

I am having this same problem. It says in the docs to listen for events for error handling but I cannot get Laravel to see any of the events coming from this package.
